doc/src/guides/solving/solve-numerically.md

```diff
@@ -0,0 +1,225 @@
+# Solve a system of equations numerically
+
+Use SymPy to numerically solve a system of one or more equations. For example,
+numerically solving $\cos(x) = x $ returns $ x \approx 0.739085133215161$.
+
+Alternatives to consider:
+- [NumPy](https://numpy.org/doc/stable/reference/generated/numpy.linalg.solve.html?highlight=solve#numpy.linalg.solve)
+and
+[SciPy](https://docs.scipy.org/doc/scipy/reference/generated/scipy.linalg.solve.html#scipy.linalg.solve)
+can each solve a system of linear scalar equations
+- To solve an underdetermined system of linear equations, convert them to
+  matrices and solve using [`gauss_jordan_solve`](gauss_jordan_solve).
+
+Here is a simple example of numerically solving one equation:
+
+```py
+>>> from sympy import cos, nsolve, Symbol
+>>> x = Symbol('x')
+>>> nsolve(cos(x) - x, x, 1)
+0.739085133215161
+```
+
+{func}`~.nsolve` calls, and can pass parameters to,
+[mpmath.findroot](https://mpmath.org/doc/current/calculus/optimization.html#root-finding-findroot).
```

Comment:
We should use intersphinx (see how it is done currently in conf.py for matplotlib, and then reference it with `{external}`).